gauntlet/packages/gauntlet-solana-contracts/src/lib/rdd.ts

```diff
@@ -22,8 +30,24 @@ function loadAggregator(contractAddress: string, network?: string, rddPath?: str
   return aggregator
 }
 
+function getContractFromRDD(rdd: any, address: string) {
```

Comment:
Please add a TODO / Issue that highlights this repeated code. It's a sign that we should bring this to a common place, not sure where yet